package.json

```diff
@@ -35,5 +35,8 @@
   },
   "devEngines": {
     "node": "8.5 || 9.x"
+  },
+  "jest": {
+    "testURL": "http://localhost"
```

Comment:
Why this change?

Reply:
@bvaughn: I was getting this runtime error (running the test suite without this change) =>

```
> jest test.js

 FAIL  ./test.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins
      
      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
          at Array.forEach (<anonymous>)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.824s, estimated 4s
```

I've actually seen this error when using Jest in other projects. The solution I've come across (fixing the exception) is discussed in https://github.com/facebook/jest/issues/6766#issuecomment-408344243.